great_expectations/data_context/store/validation_config_store.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+from great_expectations.core.data_context_key import DataContextKey, StringKey
+from great_expectations.core.validation_config import ValidationConfig
+from great_expectations.data_context.cloud_constants import GXCloudRESTResource
+from great_expectations.data_context.store.store import Store
+from great_expectations.data_context.types.resource_identifiers import (
+    GXCloudIdentifier,
+)
+
+
+class ValidationConfigStore(Store):
+    _key_class = StringKey
+
+    def __init__(
+        self,
+        store_name: str,
+        store_backend: dict | None = None,
+        runtime_environment: dict | None = None,
+    ) -> None:
+        super().__init__(
+            store_name=store_name,
+            store_backend=store_backend,
+            runtime_environment=runtime_environment,
+        )
+
+    def get_key(
```

Comment:
Implementing the bare minimum required in the factory. Tests should reflect this as well